dashboard/app/models/lesson.rb

```diff
@@ -267,6 +267,14 @@ def lesson_plan_base_url
     CDO.code_org_url "/curriculum/#{script.name}/#{relative_position}"
   end
 
+  def unit_standards_url
+    CDO.code_org_url "/s/#{script.name}/standards"
+  end
+
+  def unit_vocabulary_url
+    CDO.code_org_url "/s/#{script.name}/vocab"
```

Comment:
please use rails path helper. you shouldn't need to define the `unit_vocabulary_url` method (or the one above it), you can just use the rails path helper directly. here's how i got the path helper for standards. you should be able to repeat the process for vocab:
```
[development] dashboard > include Rails.application.routes.url_helpers
=> Object
[development] dashboard > Rails.application.routes.named_routes.helper_names.select {|name| name.include?('standard')}
=>
["standards_course_path",
 "search_standards_path",
 "standards_script_path",
 "dashboardapi_script_standards_path",
 "api_script_standards_path",
 "standards_course_url",
 "search_standards_url",
 "standards_script_url",
 "dashboardapi_script_standards_url",
 "api_script_standards_url"]
[development] dashboard > unit = Unit.find_by_name('csd3-2023')
=>
#<Unit:0x00007fdb202b8880
...
[development] dashboard > standards_script_path(unit)
=> "/s/csd3-2023/standards"
```